common.go

```diff
@@ -969,18 +992,22 @@ var writerMutex sync.Mutex
 // A Certificate is a chain of one or more certificates, leaf first.
 type Certificate struct {
 	Certificate [][]byte
+
```

Comment:
We should keep the diff as conservative as possible, so I would remove these whitespace changes along with others in other files

Reply:
Ack.

Reply:
Done.